### ISSUE TITLE
Reading parameters from `toml` in the yaml file assumes that the toml is in the working directory

### ISSUE THREAD

**Issue by Sbozzolo** [CliMA/ClimaAtmos.jl#2167]
Steps to reproduce:
```julia
import ClimaAtmos as CA
cd("/tmp")
CA.AtmosConfig(; config_dict = CA.config_from_target_job("sphere_aquaplanet_rhoe_equilmoist_allsky_gw_raw_zonallyasymmetric"))
```
Returns this errror:
```
"toml/sphere_aquaplanet_rhoe_equilmoist_allsky_gw_raw_zonallyasymmetric.toml": No such file
```
As a result, it is not practical to run the various configurations from anywhere but the root of the `ClimaAtmos` project.

I am not sure what is the cleanest way to fix this. On one side, reading the toml distributed with `ClimaAtmos` defies the purpose of having them as a configuration. On the other hand, this makes impossible to run the configurations unless one clones the `ClimaAtmos` repo and runs the code from the root of the project.

**Comment by nefrathenrici** [comment]
The simplest fix I have come up with: if the TOML path is not a valid file, we can prepend the default toml location to the path, check if that is a file, and notify the user. This would support both:
- users submitting their own parameter files that aren't in the default `/toml/` location
- users running the given model configurations outside of the who want to access the default TOML files.

**Comment by Sbozzolo** [comment]
I think that this would cover most of the common use cases I can think of. (A case that would be not be covered if someone wants to run an example configuration but change one single parameter.)

**Comment by nefrathenrici** [comment]
> A case that would be not be covered if someone wants to run an example configuration but change one single parameter.

Yes, this would require a bigger change, and it is not too difficult in the current setup:
```
config_dict = CA.config_from_target_job("sphere_aquaplanet_rhoe_equilmoist_allsky_gw_raw_zonallyasymmetric")
config_dict["toml"]  = "path/to/toml"  
# Or config_dict["toml"] = Dict
CA.AtmosConfig(; config_dict)
```

**Comment by simonbyrne** [comment]
First step is to document all the input files required.
